src/main/java/control/ControllerManager.java

```diff
@@ -30,11 +31,13 @@
     private ShotBlock activeShotBlock;
 
     // Placeholder project in lieu of XML loading
-    @Getter @Setter
+    @Getter
+    @Setter
     private ScriptingProject scriptingProject = new ScriptingProject("BOSS Project", 1.0);
 
     /**
      * Constructor.
+     *
```

Comment:
This is on purpose for readability of the comments. It doesn't add much in this case but I want to be consistent